### PR TITLE
feat:Added a new policy for VMInstance.

### PR DESCRIPTION
**Pull request by kushZemoso**
This rule checks if serial-port for vminstance is disabled.

# Description
Ensure that "Enable connecting to serial ports" configuration setting is disabled for all your production Google Compute Engine instances.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also
list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] The commit message/PR follows our guidelines

# **Other information**:

### CODE REVIEW

File: jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/gcprules/vminstance/VMInstanceSerialPortDisableRule.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+                throw new RuleExecutionFailedExeption(exception.getMessage());
+            }
+        }
+
+        logger.debug("========VMInstanceSerialPortDisableRule ended=========");
+        return new RuleResult(PacmanSdkConstants.STATUS_SUCCESS, PacmanRuleConstants.SUCCESS_MESSAGE);
+
+    }
+
+    private boolean verifyIsBlockProjectWideSSHEnable(String vmEsURL, Map<String, Object> mustFilter) throws Exception {
```
Comment (dheerajkholia-paladin) [inline jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/gcprules/vminstance/VMInstanceSerialPortDisableRule.java:90]: Can we rename the method relevant to the policy name .

File: installer/resources/pacbot_app/files/DB.sql [hunk 2]
```diff
@@ -2021,6 +2024,8 @@ INSERT IGNORE INTO cf_RuleInstance (ruleId,ruleUUID,policyId,ruleName,targetType
...
+
```
Comment (dheerajkholia-paladin) [inline installer/resources/pacbot_app/files/DB.sql:2028]: Please resolve conflicts on this file

Comment (kushZemoso) [reply installer/resources/pacbot_app/files/DB.sql:2028]: resolved conflict.

File: jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/gcprules/vminstance/VMInstanceSerialPortDisableRule.java (hunk 1 above)
Comment (kushZemoso) [reply jobs/pacman-awsrules/src/main/java/com/tmobile/cloud/gcprules/vminstance/VMInstanceSerialPortDisableRule.java]: changed method name to verifyIsSerialPortDisable( ).

**Review by dheerajkholia-paladin — APPROVED** [review]

**Pull request merged**